integrations/vip-composable-blocks/airtable.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Integration: Airtable.
+ *
+ * @package Automattic\VIP\Integrations
+ */
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads Airtable Integration.
+ *
+ * @private
+ */
+class AirtableIntegration extends Integration {
```

Comment:
Do we need to enumerate the different types of child integrations here? Or can a single generic datasource integration suffice? Ideally we don't need to release mu-plugins in order to add a new datasource type.

Reply:
Addressed in [5ae46f4](https://github.com/Automattic/vip-go-mu-plugins/pull/5644/commits/5ae46f4b0d1fd68da0c5593843313804905247c7)

Reply:
> Ideally we don't need to release mu-plugins in order to add a new datasource type.

As per current implementation we do need to release MU Plugins in order to add a new data source because the data_sources are hard coded in `VipComposableBlocksIntegration`.

If this is a must have requirement then we can update ConfigMap template to include `parent_type` attribute and then get data_sources based on this attribute. Please do let me know about your thoughts on this.

Reply:
Fine with me to hardcode possible data sources as long as we have a pain-free path to not hardcode it in the future

Reply:
Okay so for now I am going with hard coded approach. In future we can update the implementation if needed.